tools/internal_ci/macos/grpc_interop_toprod.sh

```diff
@@ -18,8 +18,8 @@ set -ex
 # change to grpc repo root
 cd $(dirname $0)/../../..
 
-source tools/internal_ci/helper_scripts/prepare_build_macos_interop_rc
 source tools/internal_ci/helper_scripts/prepare_build_macos_rc
+source tools/internal_ci/helper_scripts/prepare_build_macos_interop_rc
```

Comment:
reordered to stay in sync with the interop scripts on linux. Also makes more sense this way.